Gekko/TradingPlatforms/BTCTradeUA/BTCTradeUATradingPlatform.swift

```diff
@@ -254,7 +254,9 @@ class BTCTradeUATradingPlatform : TradingPlatform {
                                                   Currency.DASH,
                                                   Currency.ZEC,
                                                   Currency.BCH,
-                                                  Currency.ETC]
+                                                  Currency.ETC,
+                                                  Currency.KRB,
```

Comment:
Please move these currencies after LTC. They should be one of the first in the list